scripts/overrides/tei_to_es_other.rb

```diff
@@ -43,6 +43,7 @@ def uri
     "#{@options["site_url"]}/published/LG/#{@year}/whole.html"
   end
 
+<<<<<<< HEAD
```

Comment:
Need to watch what you're committing more carefully. Don't want to commit the merge conflict outlines here. Recommend making a habit of looking at a git GUI to quickly see what the git diff is before making every commit. I especially always review before I make commits when I've been working on something more than a tiny fix I just made after other commits or having checked there were no other unmerged changes.

This is also an example of stuff that would ideally be caught before posting a PR and fixed with history revision.

Reply:
The TEI/XSL commit (made by Karin) is already on dev, I was just bringing it in so I could start working on some Nokogiri transformations.

Reply:
If it's already on dev then you should rebase your branch on top of dev rather than pull those commits into your branch.

Reply:
A more precise account of that commit. I noticed there were a lot of "duplicate commits" on this branch, apparently because I had previously rebased on dev. Commits authored by both me and someone else. Is this what you mean by duplicate commits? I think they happen when the `git rebase dev` runs into conflicts and I resolve them. When I recreated this branch, I did a `git rebase -i` through all the commits and then dropped most of the commits that came from dev and had multiple authors. I forgot about the last commit by Karin, I had originally intended to remove it and then cherry-pick it. I decided to just leave it in since I needed it.

Anyway, it seems to me that rebasing (at least the way I was doing it) is what is causing this recurring problem. `git rebase dev` is not resulting in my branch applying cleanly over dev.

Reply:
Yeah, those commits that show multiple authors are usually commits that were already on another branch. I don't think that should be happening when you rebase on `dev` though. If rebase runs into a conflict it doesn't pull commits from the branch being rebased onto. It will ask you to modify the commit on your branch so it doesn't conflict, not add more commits. So I'm skeptical the rebasing is what's causing this. I think there were merges between branches before rebasing that are where they came from. You'll have to show me what you're seeing and doing next time you see a conflict and maybe we can get to the bottom of this :crossed_fingers:

Reply:
I tried the rebase, it seems like all the multiple authored commits are gone now.

Reply:
Yeah, after double-checking the commits that were removed I think we're all set now.